kythe/go/serving/xrefs/xrefs.go

```diff
@@ -93,22 +93,67 @@ type SplitTable struct {
 
 	// Documentation is a table of srvpb.Documents keyed by their node ticket.
 	Documentation table.Proto
+
+	// EdgeFilter is an optional callback to rewrite edge labels.
+	// It will be called once per request; the function it returns will then be
+	// called once per edge.
+	EdgeFilter func(context.Context) func(*string)
```

Comment:
nit: this is more of a map than a filter

Reply:
Could just call it "RewriteEdgeLabel", since that's what it does. 🙂